Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -888,8 +886,6 @@ PhysicalParticleContainer::FieldGather (int lev,
         {
             Real wt = amrex::second();
 
-            const Box& box = pti.validbox();
-
```

Comment:
Fixes the warning:
`./Source/Particles/PhysicalParticleContainer.cpp:875:24`: warning: unused variable ‘box’ [-Wunused-variable]
             const Box& box = pti.validbox();